dev/ci/c-bedrock2.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+bedrock2_CI_DIR="${CI_BUILD_DIR}/bedrock2-ci"
+
+git_checkout "${bedrock2_CI_BRANCH}" "${bedrock2_CI_GITURL}" "${bedrock2_CI_DIR}"
+( cd "${bedrock2_CI_DIR}" && git submodule update --init --recursive )
+
+( cd "${bedrock2_CI_DIR}" && env EXPECTED_COQC_VERSION="$(coqc --version | sed -n 's/The Coq Proof Assistant, version \([^ ]*\) .*/\1/p')" make )
```

Comment:
This is a hack, can't we get upstream support for it instead?

Reply:
I also think that this is a hack and I'll remove the `EXPECTED_COQC_VERSION` check from bedrock2.